test/testaudio.c

```diff
@@ -581,6 +593,13 @@ static void StreamThing_ondrop(Thing *thing, int button, float x, float y)
     }
 }
 
+static void StreamThing_onmousewheel(Thing *thing, float y)
+{
+    thing->meter += y * 0.01f;
+    thing->meter = SDL_clamp(thing->meter, 0.0f, 1.0f);
```

Comment:
Not requesting a change or anything, but it seems silly that this is called gain and is never gain, it's always attenuation.

Reply:
I clamped it for the test app, but one _can_ set it to 2.0f to double the volume.